src/test/java/com/google/api/generator/gapic/dummy/FileDiffInfraDummyTest.java

```diff
@@ -63,9 +66,23 @@ public void classWithHeader() {
             .build();
     JavaWriterVisitor visitor = new JavaWriterVisitor();
     classDef.accept(visitor);
-    assertEquals(visitor.write(), EXPECTED_CLASS_STRING_WITH_HEADER);
+    Path goldeFilePath =
+        Paths.get(GOLDENFILES_DIRECTORY, "FileDiffInfraDummyTestClassWithHeader.golden");
+    Assert.assertCodeEquals(goldeFilePath, visitor.write());
+  }
+
+  // Add a simple test for two strings comparison.
+  @Test
+  public void simpleLineComment() {
+    JavaWriterVisitor visitor = new JavaWriterVisitor();
+    LineComment lineComment = LineComment.withComment("test strings comparison.");
+    lineComment.accept(visitor);
+    Assert.assertCodeEquals("// test strings comparison.", visitor.write());
   }
 
+  private static final String GOLDENFILES_DIRECTORY =
```

Comment:
Nit: Could we please move this to the top? The Apache License is an exception; since it's "large clutter", putting it at the bottom makes the test more readable.

Reply:
Sounds good to me, moved it to top, thanks!